lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -224,10 +248,15 @@ class ItemListPresenter {
                 .map { _ in return () }
                 .asDriver(onErrorJustReturn: ())
 
-        self.dataStore.syncState
-                .asDriver(onErrorJustReturn: SyncState.Synced)
+        Observable.combineLatest(self.dataStore.syncState, self.itemListDisplayStore.listDisplay)
```

Comment:
Similarly, you can use `isManualRefreshObserver` which you've already filtered for previously to avoid the `if let` in the drive section.